src/expressions/operator-expr.md

```diff
@@ -366,9 +366,8 @@ same trait object.
         * zero-extend if the source is unsigned
         * sign-extend if the source is signed
     * Casting from a float to an integer will round the float towards zero
-        * **[NOTE: currently this will cause Undefined Behavior if the rounded
-          value cannot be represented by the target integer type][float-int]**.
-          This includes Inf and NaN. This is a bug and will be fixed.
+        * `NaN` will return `0` and values outside the integer range will
+          saturate to the integer minimum/maximum.
```

Comment:
We should state this more explicitly that values smaller than the integer range will saturate to the minimum value of the integer type and that values larger will saturate towards the maximum value of the integer type. Each one in its own bullet point.

I can do this as a follow-up PR if you don't want to.